include/swift/AST/DiagnosticsSema.def

```diff
@@ -2703,8 +2703,8 @@ NOTE(decl_import_via_here,none,
 NOTE(decl_import_via_local,none,
      "%kind0 is imported by this file as "
      "'%select{private|fileprivate|internal|package|%ERROR|%ERROR}1' "
-     "from %2",
-     (const Decl *, AccessLevel, const ModuleDecl*))
+     "from %select{%2|bridging header}3",
```

Comment:
I feel like there may be more resilience diagnostics that will need to be updated, but it seems like this is a good start

Reply:
Perhaps! They'll come out as the weird __ObjC module if we don't customize them.

Reply:
Ah, that could help here, yes.

Reply:
I did this in https://github.com/swiftlang/swift/pull/84410/commits/b7a83495fa58b927f05346161ac67760f5356779